ui/src/App.vue

```diff
@@ -34,11 +35,15 @@ export default {
       modules: [{ text: "loading..." }]
     };
   },
-
   created: function() {
-  },
-
-  mounted: function() {}
+    window.gapi.load('auth2', initAuth);
+    function initAuth() {
```

Comment:
Since this is the main App, maybe we could consider making the function `async initAuth()` so it won't block other loads, as an optimization later?